UPGRADING.md

```diff
@@ -4,6 +4,18 @@ Whilst experimenting with the API & benefiting from changes in Ember, we've had
 
 Here are the details on updating from previous versions.
 
+## From v2.0.0
+
+### Deprecations
+1. You will not longer be able to import `CanMixin` as it will be removed. You can reproduce its behavior using `can` service and its methods.
+2. `import { CanService } from 'ember-can'` -> `import CanService from 'ember-can/services/can'`
+3. `import { ability } from 'ember-can'` -> `import { ability } from 'ember-can/computed'`
+4. `build` method of `can` service will be replaced by `abilityFor`
+
+### Notable changes
+1. You have to pass a `null` model if you want to pass only parameters to `can` or `cannot` methods of `can` service. Eg: `this.get('canService').can('post', null, { test: true})`
+2. The `model` passed to helpers or `can` service methods, as from now on, can be anything.
```

Comment:
I wasn't aware that there were any restrictions earlier, to be honest. Looking at the changes it seems like we could only use Ember.Object instances before. Perhaps we should make that clear using:
> The `model` passed to helpers or `can` service methods, as from now on, can be anything instead of an `Ember.Object` instance.

This isn't super important, by any means. I'll leave this to your consideration.

Again, great job on this.